code/game/objects/effects/effect_system/effects_foam.dm

```diff
@@ -119,7 +119,7 @@
 	STOP_PROCESSING(SSfastprocess, src)
 	if(metal)
 		var/turf/T = get_turf(src)
-		if(isspaceturf(T)) //Block up any exposed space
+		if(isspaceturf(T) || istype(T, /turf/open/openspace)) //Block up any exposed space
```

Comment:
Please add an "NSV13" tag to this change since it's in the base code

Reply:
done